client/Makefile

```diff
@@ -12,4 +12,6 @@ micromini-tool.completions: micromini-tool
 micromini-tool: micromini-tool.c
 	$(CC) -Wall -o $@ -g -Os $^
 
-
+service-install:
```

Comment:
make micromini-tool a prereq for this?

Reply:
like this?

Reply:
Also added the service to all.

Reply:
yep! that way you can't service-install without compiling the executable